### PR TITLE
Simplified m2m_recursive.tests.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Comment by carltongibson** [comment]
Hey @felixxm. 

Diff looks nice, but there's a test error (on MySQL/MariaDB, SQLite by the looks of it):

```
$ ./runtests.py m2m_recursive
...
======================================================================
ERROR: setUpClass (m2m_recursive.tests.RecursiveM2MTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  ...
  File "django/db/models/fields/related_descriptors.py", line 840, in __init__
    (instance, self.pk_field_names[self.source_field_name]))
ValueError: "<Person: Anne>" needs to have a value for field "id" before this many-to-many relationship can be used.
```

**Comment by carltongibson** [comment]
From the [bulk_create docs](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#django.db.models.query.QuerySet.bulk_create): 

> If the model’s primary key is an AutoField it does not retrieve and set the primary key attribute, as save() does, unless the database backend supports it (currently PostgreSQL).

File: tests/m2m_recursive/tests.py [hunk 1]
```diff
@@ -8,175 +6,55 @@
 class RecursiveM2MTests(TestCase):
     @classmethod
     def setUpTestData(cls):
-        cls.a, cls.b, cls.c, cls.d = [
-            Person.objects.create(name=name)
-            for name in ["Anne", "Bill", "Chuck", "David"]
-        ]
-
-        # Anne is friends with Bill and Chuck
+        cls.a, cls.b, cls.c, cls.d = Person.objects.bulk_create([
```
Comment (carltongibson) [inline tests/m2m_recursive/tests.py]: Just reverting this chunk should be sufficient.

**Comment by felixxm** [comment]
@carltongibson Thanks! Ahh, I wanted to be too clever :wink:

**Pull request merged**